OWML.Launcher/App.cs

```diff
@@ -70,6 +72,64 @@ private void LocateGamePath()
             }
         }
 
+        private void CheckGameVersion()
+        {
+            var versionReader = new GameVersionReader(_writer, new BinaryPatcher(_owmlConfig, _writer));
+            var gameVersion = versionReader.GetGameVersion();
+            _writer.WriteLine($"Game version: {gameVersion}", MessageType.Info);
+            var splitGameVersion = gameVersion.Split('.');
+            var splitMinVersion = _owmlManifest.MinimalGameVersion.Split('.');
+            var splitMaxVersion = _owmlManifest.MaximalGameVersion.Split('.');
+            var count = Math.Min(splitGameVersion.Length, Math.Min(splitMaxVersion.Length, splitMinVersion.Length));
+            splitGameVersion[count - 1] = string.Join("", splitGameVersion.Skip(count - 1));
+            splitMinVersion[count - 1] = string.Join("", splitMinVersion.Skip(count - 1));
+            splitMaxVersion[count - 1] = string.Join("", splitMaxVersion.Skip(count - 1));
+            for (var i = 0; i < count; i++)
+            {
+                var gameNumber = Convert.ToInt32(splitGameVersion[i]);
+                var minNumber = Convert.ToInt32(splitMinVersion[i]);
+                var maxNumber = Convert.ToInt32(splitMaxVersion[i]);
+                if (gameNumber < minNumber)
+                {
+                    ThrowBelowError();
+                }
+                if (gameNumber > maxNumber)
+                {
+                    ThrowAboveWarning(gameVersion);
+                    return;
+                }
+            }
+        }
+
+        private void ThrowBelowError()
+        {
+            _writer.WriteLine("Unsupported game version found", MessageType.Error);
+            _writer.WriteLine("Press any key to exit...", MessageType.Info);
+            Console.ReadKey();
+            ExitConsole();
+        }
+
+        private void ThrowAboveWarning(string currentVersion)
+        {
+            _writer.WriteLine("Potentially unsupported game version found", MessageType.Warning);
```

Comment:
Since it's *potentially* unsupported, I don't think we should bother user with a prompt. I say we print a warning and continue. Keep it simple :)